src/widgets/dialogs/settings/generalpage.cpp

```diff
@@ -55,6 +56,17 @@ void GeneralPage::setupUI()
                 this, &GeneralPage::pageIsChanged);
     }
 #endif
+
+#if not defined(Q_OS_MAC)
```

Comment:
According to the manual, we could use `Q_OS_MACOS`, which is enough.

```suggestion
#if not defined(Q_OS_MACOS)
```